### PR TITLE
Issue 1206

### PR DESCRIPTION
**Pull request by AbfrmBlr**

### CODE REVIEW

**Comment by AbfrmBlr** [comment]
test this please

**Comment by AbfrmBlr** [comment]
test this please

File: clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java [hunk 1]
```diff
@@ -204,7 +168,7 @@
    */
   // This method is intended for unit test use; modifications are likely needed for other (monitoring) purposes
   Set<String> getStores() {
-    return Collections.unmodifiableSet(new HashSet<String>(stores.keySet()));
+    return Collections.unmodifiableSet(new HashSet<String>(serverStoreManager.getStores().keySet()));
```
Comment (ljacomet) [inline clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java]: With the changes, does it still make sense to keep these "unit test" methods here?

Comment (AbfrmBlr) [inline clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java]: You are right they don't belong here. Actually they will be moved to actual test cases of the service emerging out of #1207, and then these methods will be gone from entity.

**Pull request merged**